wp-content/themes/bzLLKit/functions.php

```diff
@@ -353,7 +353,7 @@ function bz_register_activity() {
 		'show_in_admin_bar'     => true,
 		'show_in_nav_menus'     => true,
 		'can_export'            => true,
-		'has_archive'           => true,		
+		'has_archive'           => true,
```

Comment:
so the space here looks like a tab or something was at the end of the line and your change just cleaned that up. nothing to worry about

Reply:
btw it was prolly done automatically when you hit save lol